build-system/tasks/release/index.js

```diff
@@ -176,78 +195,72 @@ function discoverDistFlavors_() {
 /**
  * Compiles all AMP flavors sequentially.
  *
- * @param {!Array<!Object>} distFlavors list of AMP flavors to build.
+ * @param {string} flavorType AMP flavor to build.
+ * @param {string} command `amp` command to build the flavor.
  * @param {string} tempDir full directory path to temporary working directory.
  */
-async function compileDistFlavors_(distFlavors, tempDir) {
-  for (const {flavorType, command: baseCommand} of distFlavors) {
-    // TODO(danielrozenberg): remove undefined case when the release automation platform explicitly handles it.
-    const command =
-      argv.esm === undefined
-        ? `${baseCommand} --esm && ${baseCommand}`
-        : argv.esm
-        ? `${baseCommand} --esm`
-        : baseCommand;
-    log('Compiling flavor', green(flavorType), 'using', cyan(command));
-
-    execOrDie('amp clean');
-    execOrDie(command);
-
-    const flavorTempDistDir = path.join(tempDir, flavorType);
-    log('Moving build artifacts to', `${cyan(flavorTempDistDir)}...`);
-    await fs.ensureDir(flavorTempDistDir);
-
-    await Promise.all(
-      DIST_DIRS.map((distDir) =>
-        fs.move(distDir, path.join(flavorTempDistDir, distDir))
+async function compileDistFlavors_(flavorType, command, tempDir) {
+  // TODO(danielrozenberg): remove undefined case when the release automation platform explicitly handles it.
+  if (argv.esm === undefined) {
+    command = `${command} --esm && ${command}`;
+  } else if (argv.esm) {
+    command += ' --esm';
+  }
```

Comment:
From my reading of release automation code, `ESM_FLAG` is set to one of `('--esm', '--no-esm')` , and this task is invoked as:
```py
amp release --output_dir "${EXTERNAL_CONTENT_DIR}" --flavor "${FLAVOR}" "${ESM_FLAG}"
```

Does this mean the `argv.esm === undefined` case here can be removed?